src/benefits/benefits.service.ts

```diff
@@ -89,7 +89,7 @@ export class BenefitsService {
     const headers = {
       Accept: 'application/json',
       'Content-Type': 'application/json',
-      Authorization: req.headers['authorization'] ?? req.headers['Authorization'],
+      Authorization: authToken,
```

Comment:
why are we passing tokens from controllers to services?
when service can directly access it as
`const authToken = getAuthToken(req);` ?